lib/shopify-cli/commands/populate/resource.rb

```diff
@@ -20,30 +20,33 @@ def call(args, command_name)
         end
 
         def call(args, _)
-          @args = args
-          @input = Hash.new
-          @count = DEFAULT_COUNT
-          @help = false
-          input_options
-          resource_options.parse(@args)
-
-          if @help
-            output = Populate.extended_help
-            output += "\n{{bold:{{cyan:#{resource_type.capitalize}}} options:}}\n"
-            output += resource_options.help
-            return @ctx.page(output)
-          end
+          if Project.at(Dir.pwd)
```

Comment:
What about `return unless Project.at(Dir.pwd)` instead and then the indentation can stay?